python/tests/__snapshots__/test_translator.ambr

```diff
@@ -0,0 +1,137 @@
+# serializer version: 1
```

Comment:
What is this file? I don't see it mentioned anywhere, but maybe you have some clever test setup that uses this?

Reply:
Yeah, [syrupy](https://github.com/tophat/syrupy) is a pytest plugin that makes it easy-ish to generate snapshot/baseline tests. They have a default snapshot format called Amber (which I've never heard of)